lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runtime/azbatch/util/batch/AzureBatchHelper.java

```diff
@@ -90,19 +111,47 @@ public void submitJob(final String applicationId, final String storageContainerS
         .withName(SharedAccessSignatureCloudBlobClientProvider.AZURE_STORAGE_CONTAINER_SAS_TOKEN_ENV)
         .withValue(storageContainerSAS);
 
+
+    String portMappings = "";
+
+    Iterator<Integer> iterator = this.portProvider.iterator();
+    while (iterator.hasNext()) {
+      Integer port = iterator.next();
+      System.out.println("iter port is " + port);
```

Comment:
Use log instead of println.

Reply:
Removed line. It was for my entertainment purposes only :smile: 

---
In reply to: [196513436](https://github.com/sharathmalladi/reef/pull/5#discussion_r196513436) [](ancestors = 196513436)